### ISSUE TITLE
ERR_UNSAFE_PORT -- what's happening here? InstantWP won't load past the main control panel

### ISSUE THREAD

**Issue by CDMacalister** [corvideon/InstantWP#103]
Okay, so out of the blue, when now when I try to login to my local WordPress site from the InstantWP control panel, I get this --

#This site can't be reached
The webpage at http://127.0.0.1:10080/wordpress/wp-login.php might be temporarily down or it may have moved permanently to a new web address.
ERR_UNSAFE_PORT

It is not the particular browser, because I get an error in any browser I use.

I have already done the command line restarts of Apache2 and MariadB, and that's not working.

I tried installing a completely fresh new InstantWP with original settings. The same error arises.

And all of the advice provided in the troubleshooting offers catch-22s: in order to try the solution requires the problem not exist in the first place.

How do I fix this? I had a lot of progress on the drive, I need this solved and solved as soon as possible.

**Comment by Rendt** [comment]
Have you found a fix? I encountered the same results with a fresh installation on Windows 10.

**Comment by ecations** [comment]
Same problem here. It used to work but now both Chrome and Firefox complain about an unsafe port..

**Comment by ecations** [comment]
I managed to find a solution using firefox and the network.security.ports.banned.override command. First I tried to change the PortOffset parameter from IWP but for some reason that didn't work for me. I outlined the solution in more detail here:
https://www.oneminutevideotutorials.com/2021/06/21/how-to-keep-using-port-10080-after-many-browsers-block-it-as-an-unsafe-port/

**Comment by essexgit** [comment]
> I managed to find a solution using firefox and the network.security.ports.banned.override command. First I tried to change the PortOffset parameter from IWP but for some reason that didn't work for me. I outlined the solution in more detail here:
> https://www.oneminutevideotutorials.com/2021/06/21/how-to-keep-using-port-10080-after-many-browsers-block-it-as-an-unsafe-port/

Any joy yet?
I tried several offset values (8000, 20000) the error changes to ERR_CONNECTION_REFUSED.

**Comment by popsy-blue** [comment]
Some ports are blocked by the browsers (same happening with Safari).

Workaround:
1. Choose a different offset (e.g. 24999) and calculate the HTTP Port as offset + 80 (e.g. 25079)
2. Start InstantWP
3. Log into MySQL Admin (from the InstantWP console)
4. Open in the "wordpress" database the table "wp_options"
5. Change the values of "siteurl" and "home" to show the calculated port (e.g. 25079 instead of 10080)
6. Re-Start InstantWP

**Comment by nullandvoidsky** [comment]
I had the same problem and I've found an easy solution.  THIS WORKS!

I use chocolatey (in Windows PowerShell) to install and maintain packages on my Windows 10 system.
[https://chocolatey.org/install](url)

I suspected that the issue was related to permissions, so after messing around with various solutions that were suggested online and still having the issue... I wen't back to the start, uninstalled instantwordpress, and installed it again... then, instead of running the Start-InstantWP.bat from anywhere else I stayed in my Powershell console window (Run as Administrator)... changed to the appropriate directory, and ran Start-InstantWP.bat 

It appears as though the administrative permissions that I used to install InstantWP in the Powershell console, were also what Start-InstantWP.bat needed to set the correct permissions for all my browsers to be able to access 127.0.0.1:10080 without any problems.

![Screenshot (453)](https://user-images.githubusercontent.com/55648289/165087221-db665a33-b5bd-455a-a5e0-4ea910879f63.png)
![Screenshot (454)](https://user-images.githubusercontent.com/55648289/165087226-9b6799df-8060-4c9e-b4ce-9c554e484f35.png)

**Comment by J8cob1** [comment]
> Some ports are blocked by the browsers (same happening with Safari).
> 
> Workaround:
> 
>     1. Choose a different offset (e.g. 24999) and calculate the HTTP Port as offset + 80 (e.g. 25079)
> 
>     2. Start InstantWP
> 
>     3. Log into MySQL Admin (from the InstantWP console)
> 
>     4. Open in the "wordpress" database the table "wp_options"
> 
>     5. Change the values of "siteurl" and "home" to show the calculated port (e.g. 25079 instead of 10080)
> 
>     6. Re-Start InstantWP

When applying @popsy-blue 's solution, don't change the portOffset in the config file at first. Open up the port as described in the article @ecations mentioned (https://www.oneminutevideotutorials.com/2021/06/21/how-to-keep-using-port-10080-after-many-browsers-block-it-as-an-unsafe-port/) so you can access MySQL Admin. Then, do the steps in the solution (you can probably skip step 6). When you are done, change the portOffset in the config file then restart.
